openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -605,6 +605,69 @@ def fill_account_journal_payment_credit_debit_account_id(env):
         )
 
 
+def create_new_counterpar_account_payment_transfer(env):
+    # Create new counterpart payment with account payment transfer
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO account_payment (move_id, is_internal_transfer, partner_type,
+            payment_type,
+            amount, currency_id,
```

Comment:
Missing `is_internal_transfer` flag (forced to True):

```suggestion
            amount, currency_id, is_internal_transfer,
```

Reply:
@pedrobaeza I added it at line 613

Reply:
Ouch, sorry. I didn't see it. The other part of the type is the most important one.